com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/GlobalLightingQualitySettings.cs

```diff
@@ -110,9 +106,9 @@ internal GlobalLightingQualitySettings()
             RTGIHalfResDenoise[(int)ScalableSettingLevelParameter.Level.Medium] = false;
             RTGIHalfResDenoise[(int)ScalableSettingLevelParameter.Level.High] = false;
 
-            RTGIDenoiserRadius[(int)ScalableSettingLevelParameter.Level.Low] = 0.75f;
-            RTGIDenoiserRadius[(int)ScalableSettingLevelParameter.Level.Medium] = 0.5f;
-            RTGIDenoiserRadius[(int)ScalableSettingLevelParameter.Level.High] = 0.25f;
+            RTGIDenoiserRadius[(int)ScalableSettingLevelParameter.Level.Low] = 1.0f;
```

Comment:
Given it is all the same value, does it make sense to keep as a scalability setting? (is it beneficial if user change this defaults or should they always be the same value?)

Reply:
Users can chose to lower it for lower perf modes it as it can affect performance a lot

Reply:
If it has an high impact, shouldn't this have different values by default too though? Out of the box the scalability does nothing here.

Reply:
To be disccused